src/preferences/dialog/dlgpreflibrarydlg.ui

```diff
@@ -132,16 +132,19 @@
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
-       <widget class="QCheckBox" name="checkBox_SyncTrackMetadataExport">
+       <widget class="QCheckBox" name="checkBox_SyncTrackMetadata">
         <property name="text">
-         <string>Automatically write modified track metadata from the library into file tags</string>
+         <string>Synchronize track metadata with file tags</string>
```

Comment:
```suggestion
         <string>Synchronize track metadata between library and track files</string>
```

Hopefully this is more clear that it is a two way feature, and the track file is altered.

Reply:
```suggestion
         <string>Synchronize track metadata from/to file tags</string>
```

Reply:
Yes, how about this:

```suggestion
         <string>Synchronize library track metadata from/to track file tags</string>
```

Reply:
https://github.com/mixxxdj/mixxx/pull/4218/commits/a33b9bb0e99bd9ee2762cc70080c1ea3364e659a